test/rpc-server/runTests.py

```diff
@@ -107,20 +107,23 @@ def checkGolden (resp, resp_golden_path):
 
 def runTest(def_path, req, resp_golden_path):
     with subprocess.Popen(f"{SERVER} {def_path} --module TEST --server-port {PORT} --log-level {server_log_level[VERBOSITY]}".split()) as process:
-        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
```

Comment:
Maybe we should rewrite this to run a whole array of tests on the same server? (does not have to be in this PR, obviously)